lib/config.js

```diff
@@ -20,14 +22,51 @@ function Config () {
 	for (var key in config) {
 		this[key] = config[key];
 	}
+	
+	if (mode === 'worker'){
+		if (!this.output_root_dir) {
+			this.output_root_dir = '~/Guerilla';
+		}
+		this.output_root_dir = this.expandTilde(this.output_root_dir);
+		mkdirp.sync(this.output_root_dir);
```

Comment:
Use this instead: https://github.com/jprichardson/node-fs-extra#mkdirsdir-callback